### ISSUE TITLE
Paper’s  R vs config negative_num / negative_weight

### ISSUE THREAD

**Issue by atesnrllh** [xue-pai/UltraGCN#43]
**Title:** Paper’s (R) vs config `negative_num` / `negative_weight`

Hi! In the paper you set negative sampling ratio (R=300). 
But in the released `.ini` configs I see e.g. Yelp: `negative_num=800`, `negative_weight=300` (and others like `negative_num=1500`). `neg_scores` shape is `(batch, negative_num)`, so `negative_num` looks like (R).

Also the code uses:
`loss = pos_loss + neg_loss * self.negative_weight`

Which one corresponds to the paper’s (R): `negative_num` or `negative_weight`? And why do configs use 800/1500 if (R=300)?

Thanks!
